### PR TITLE
feat: added Grid component

### PR DESCRIPTION
**Pull request by krgarima**
##Change Logs:
#Description: Added new component Grid
#Screenshot:
griditems2
griditems3
griditemsmultiple

### CODE REVIEW

**Comment by rahul31agrawal** [comment]
Looks fine to me, I had an suggestion that tha gap between the grids in 3rd example is bit more, if the gap is same as 1st two example then it will look good (optional suggestion).
Thanks

File: Components/Grid/grid.html [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+        <header>
+            <div class="navBar space-between">
+                <div class="nav-left space-evenly">
+                    <i class="fas fa-bars hamburgerMenu" id="hamburgerMenuModal"></i>
+                    <h3 class="m-r-4">Modal Drawer</h3>
+                </div>
+                <div class="nav-right space-evenly">
+                    <h3 class="m-r-4"><a href="/index.html">Get Started</a></h3>
+                    <h3 class="m-r-4"><a href="/home.html">Documentation</a></h3>
```
Comment (kirti2602) [inline Components/Grid/grid.html:25]: what's `m-r-4`? Maybe try using  self-descriptive classnames

File: Components/Grid/grid.html [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+                <div class="grid__frame center">
+                <iframe class="mt2-mb6"
+                    src="https://carbon.now.sh/embed?bg=rgba%2880%2C227%2C194%2C1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522grid-container-three%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522grid-item%2522%253E1%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522grid-item%2522%253E2%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522grid-item%2522%253E3%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+                    style="width: 500px; height: 153px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+                </div>
+
+                <h2 class="h-2 mv-4">3) Grid with multiple items</h2>
```
Comment (kirti2602) [inline Components/Grid/grid.html:117]: Here as well, didn't get what's `mv-4`

File: Components/Grid/grid.html [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+                </div>
+
+            </div>
+        </main>
+
+    </div>
+
+    <footer class="space-between">
+        <button class="nav_btn prev_page fw-1 m-4 p-1"><a href="/Components/Alert/alert.html">&lt; Alert</a></button>
+        <button class="nav_btn next_page fw-1 m-4 p-1"><a href="/Components/Badge/badge.html">Badge &gt;</a></button>
```
Comment (kirti2602) [inline Components/Grid/grid.html:145]: more descriptive class names can be used here too

**Review by kirti2602 — COMMENTED** [review]
Your page headings are very close to the paragraphs. Can have some space in between.
rest looks fine to me

File: Components/Grid/grid.html (hunk 1 above)
Comment (krgarima) [reply Components/Grid/grid.html:25]: For margin, padding, font-weight I have used variables like m-4 for margin: 4rem, m-2 for margin: 2rem, p-1 for padding: 1rem etc.

**Pull request merged**